### PR TITLE
[Core] General Fixes to make LocalNet more reliable

### PR DESCRIPTION
**Pull request by Olshansk**
## Description

I was hitting issues running LocalNet off of main, so these are just a few "bandaids" to get main to a more stable state, with some debugging improvements & validations until we handle everything properly. 

## Issue

NA

## Type of change

Please mark the relevant option(s):

- [ ] New feature, functionality or library
- [X] Bug fix
- [X] Code health or cleanup
- [ ] Major breaking change
- [ ] Documentation
- [ ] Other 

## List of changes

**General**
- Fix a few typos & added some TODOs
- Automatic edits compiled by my editor

**Makefile**
- Added `docker_wipe_nodes` to clear the DB and nodes without rebuilding everything

**Consensus**
- Make logs less noisy by avoiding returning an error when it does not need to log an actual error
- Panic if the validator configurations is incorrect
- Release the utility context when starting a new view

**P2P**
- Improve logging of targets at a certain height

**Persistence**
- Do not use the KV store when loading the genesis state
- Fix how the state is cleared 

## Testing

- [X] `make develop_test`
- [X] [LocalNet](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md) w/ all of the steps outlined in the `README`

## Required Checklist

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have tested my changes using the available tooling
- [ ] I have updated the corresponding CHANGELOG

### If Applicable Checklist
- [ ] I have updated the corresponding README(s); local and/or global
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added, or updated, [mermaid.js](https://mermaid-js.github.io) diagrams in the corresponding README(s)
- [ ] I have added, or updated, documentation and [mermaid.js](https://mermaid-js.github.io) diagrams in `shared/docs/*` if I updated `shared/*`README(s)

### CODE REVIEW

File: persistence/module.go [hunk 1]
```diff
@@ -204,19 +205,18 @@ func initializeBlockStore(blockStorePath string) (kvstore.KVStore, error) {
...
 func (m *persistenceModule) shouldHydrateGenesisDb() (bool, error) {
 	checkContext, err := m.NewReadContext(-1)
 	if err != nil {
 		return false, err
 	}
 	defer checkContext.Close()
 
-	maxHeight, err := checkContext.GetLatestBlockHeight()
-	if err == nil || maxHeight == 0 {
+	_, err = checkContext.GetLatestBlockHeight()
```
Comment (andrewnguyen22) [inline persistence/module.go:217]:
```suggestion
	if _, err = checkContext.GetLatestBlockHeight(); err != nil {
```

File: persistence/module.go [hunk 2]
```diff
@@ -204,19 +205,18 @@ func initializeBlockStore(blockStorePath string) (kvstore.KVStore, error) {
...
+	if err != nil {
```
Comment (andrewnguyen22) [inline persistence/module.go:218]:
```suggestion
```
Only if suggestion above is accepted

**Review by andrewnguyen22 — CHANGES_REQUESTED** [review]
Simple changes - just a nit

**Review by deblasis — APPROVED** [review]

File: persistence/module.go (hunk 2 above)
Comment (Olshansk) [reply persistence/module.go:218]: In the future, note that you can do a "multiline suggestion" by highlighting and dragging down multiple lines.

record

**Review by andrewnguyen22 — APPROVED** [review]

**Pull request merged**